modules/electrophysiology_browser/php/electrophysiology_browser.class.inc

```diff
@@ -0,0 +1,131 @@
+<?php
```

Comment:
```suggestion
<?php declare(strict_types=1);
```

Reply:
Done in the next commit. Should this be on top of each php script?

Reply:
Yep!